src/segmenter/dummy.example.rs

```diff
@@ -14,7 +14,7 @@ pub struct DummySegmenter;
 
```

Comment:
Why is the name of the file like so? Why not use underscores here? Like `dummy_example.rs`.